packages/dds/tree/src/feature-libraries/editable-tree/utilities.ts

```diff
@@ -86,3 +87,20 @@ export function adaptWithProxy<From extends object, To extends object>(
 export function getArrayOwnKeys(length: number): string[] {
     return Object.getOwnPropertyNames(Array.from(Array(length)));
 }
+
+const pathDelimiter = "/";
+
+function stringifyPath(path: UpPath, childPath: ETreeNodePath): ETreeNodePath {
+    const fieldName = typeof path.parentField === "string" ? path.parentField : keyFromSymbol(path.parentField);
+    return brand(`${fieldName}[${path.parentIndex}]${childPath}`);
+}
+
+export function pathToString(path: UpPath): ETreeNodePath {
+    let current = path;
+    let upPath = stringifyPath(current, brand(""));
+    while (current.parent !== undefined) {
```

Comment:
this omits the parent field when the parent is undefined.

This means which detached field the path points to is not included in the string.

This should either be changed to include it, or documented (ex:  that is assumes the root is the default root field and asserts that).

Reply:
It does not as a path to the node with an undefined parent is resolved in a line above.

Just a remark: the comments I am closing to digest the most important ones and to focus on those.